test/integration/scenarios/lib/chart-upgrade-taskfile.yaml

```diff
@@ -32,10 +32,10 @@ tasks:
     - |
       helm upgrade integration {{ .TEST_CHART_NAME }} \
         --namespace {{ .TEST_NAMESPACE }} \
-        --values {{ .TEST_VALUES_BASE_DIR }}/chart-full-setup/values-integration-test-ingress-{{ .TEST_AUTH_TYPE }}.yaml \
-        --values {{ .TEST_VALUES_BASE_DIR }}/chart-full-setup/values-integration-test-ingress-{{ .TEST_VALUES_SCENARIO }}.yaml \
         --values {{ .TEST_VALUES_BASE_DIR }}/common/values-integration-test.yaml \
         --values {{ .TEST_VALUES_BASE_DIR }}/common/values-integration-test-pull-secrets.yaml \
+        --values {{ .TEST_VALUES_BASE_DIR }}/chart-full-setup/values-integration-test-ingress-{{ .TEST_AUTH_TYPE }}.yaml \
```

Comment:
Here I correct the order as the common should come first, then overwritten with the scenario which we do in the installation task:

https://github.com/camunda/camunda-platform-helm/blob/8c6d85d5db2b9d6f242ba1ff639f23a4614b6904/test/integration/scenarios/chart-full-setup/Taskfile.yaml#L192